content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a model registry to manage the lifecycle of the models we get from
+our experiments. Using DVC and Studio we will set up a model registry where we
+can discover, share, deploy and audit all our models and which will serve as the
+single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## DVC Model registry overview
+
+In DVC Studio we can access the model registry by clicking on Models in the top
+menu. This will show you a dashboard with all models from all projects you have
+access to. You can check out our
+[public model registry example](https://studio.iterative.ai/team/Iterative/models).
+
+From the dashboard you will have an overview of all models, latest model
+versions as well stages each of the model versions is assigned to. You can also
+see which git repository for each model and get more details for it by clicking
+on the model name.
+
+Here you will see some extra information about a particular model - a
+description of the model, any labels that were assigned and particularly the
+history of all model registry actions on that selected model. For each model
+version you can also have a look at its metrics tracked by the experiment
+tracked.
+
+## Adding models
+
+Let's now train a model and add it to the model registry.
+
+We have three options how to add a model to the model registry. In this guide,
```

Comment:
I think we could drop this sentence. We show the other ways below. Let's just get into how we are doing it here.